migration/migration.go

```diff
@@ -338,6 +341,9 @@ func PopulateCommonTypes(ctx context.Context, db *gorm.DB, witr *workitem.GormWo
 	if err := createOrUpdatePlannerItemExtension(workitem.SystemFeature, ctx, witr, db); err != nil {
 		return errs.WithStack(err)
 	}
+	if err := createOrUpdatePlannerItemExtension(workitem.SystemEpic, ctx, witr, db); err != nil {
```

Comment:
The story talks about adding features in the core that would support the use case that an Epic represents, not just add an Epic.